pkg/pillar/cmd/volumemgr/handlevolumeref.go

```diff
@@ -198,35 +197,35 @@ func updateVolumeRefStatus(ctx *volumemgrContext, vs *types.VolumeStatus) {
 					status.ClearErrorWithSource()
 				}
 				publishVolumeRefStatus(ctx, status)
-				return
-			}
```

Comment:
Removing this return makes it painful to determine whether there were any changes in the following basic block or not.

Reply:
> do you need to remove it due to the possibility of multiple matches across the loop?

Exactly.

> If so, then please move the publish to after the if/else,

Done.